style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -470,6 +470,16 @@ const metadata: StyleDictionary.MetadataIndex = {
     themeable: true,
     public: true,
   },
+  colorTextLayoutToggle: {
+    description: "This is the default color of the app layout toggle button when it's active.",
+    public: true,
+    themeable: true,
+  },
+  colorTextLayoutToggleSelected: {
+    description: "This is the color of the app layout toggle button when it's selected.",
+    public: true,
+    themeable: true,
+  },
```

Comment:
Is there already a token to control the background color? Because if they change the color of the icon itself, they might need to make adjustments to the background color to reach proper color contrast, right?

Reply:
@DaemonCahill Can correct me if I'm wrong, but I believe the background color was exposed here @timogasda 

https://github.com/cloudscape-design/components/pull/1061/files#diff-bb763d60125a3154c52f870b2ed58f7b47f58dc3b2e3a25c89ba9e14cbd8933cR115-R144

Reply:
Yes, that's the one!